challenge.md

```diff
@@ -32,19 +32,21 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+
+ This is an absolute path because it is a complete path from start of the filesystem. It also is leading with a /.
+
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+We would use an absolute path becasuse the font is being hosted from a source that is not saved on your computer.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
- 
+This would be a relative path because the file would be saved on your computer.
```

Comment:
Right answer, wrong reason. Suppose that we wanted to zip up our project and give it to somebody else. If we'd used Absolute paths in our links, when this other person opened up the project, all the links would be broken, because the image file would no longer be at the same absolute address. However, if we'd used Relative paths, the links would not be broken, because the image file would still be in the same _relative position_ as it was before. This is why, as a rule of thumb, you generally want to use Relative paths for resources within your project and Absolute paths for resources outside your project.